requirements.txt

```diff
@@ -1 +1,2 @@
+isort<=4.0.0
```

Comment:
Is this used somewhere in the code itself? Or would it be better off in a `requirements-dev.txt` file?

Reply:
I'll move it to requirements-test.txt since this is used by pylint.

Reply:
Done.